snapcraft/cli/store.py

```diff
@@ -252,18 +278,91 @@ def list_registered():
     snapcraft.list_registered()
 
 
+@storecli.command('export-login')
+@click.argument('login_file', metavar='FILE', type=click.File('w'))
+@click.option('--packages', metavar='<packages>',
+              help='Comma-separated list of packages to limit token access')
+@click.option('--channels', metavar='<channels>',
+              help='Comma-separated list of channels to limit token access')
+@click.option('--acls', metavar='<acls>',
+              help='Comma-separated list of ACLs to limit token access. Valid '
+                   "ACLs are 'package_upload', 'package_access', and "
+                   "'package_manage'")
+def export_login(login_file: TextIO, packages: str, channels: str, acls: str):
+    """Save attenuated login configuration for a store account in FILE.
```

Comment:
Is a Snapcraft user expected to know what “attenuated” means in this context? This seems to be the remains of [removing the word “macaroon”](https://forum.snapcraft.io/t/better-support-for-other-ci-systems/2968/3) (since [an “attenuated macaroon” is apparently a thing](https://www.google.co.uk/search?q=%22attenuated+macaroon%22)).

Reply:
No, I suspect they won't, this is my technical knowledge infecting the help doc :stuck_out_tongue: . What do you think this should be without getting too lengthy? "Save properly-scoped login [...]"; "Save restricted login [...]"; "Save limited login [...]" any of those sound promising?

Reply:
Why is it important to know that it’s attenuated/scoped/restricted? What can’t it do?

Reply:
"limited-access login"?

Reply:
`export-login` exports a login that can be used to login (with no password) to the account that was exported. If the user provides no parameters, then whoever logs in with this file gains the full ability of the account that was exported (can access all their snaps, upload to them, publish to all channels, etc.). This is a lot of power to give a CI system. If I know that my CI only pushes to `edge` for snap `foo`, then I can limit that exported login to only have access to the `edge` channel of snap `foo` instead of everything by using `snapcraft export-login --packages foo -channels edge

Reply:
> I do think that's important to convey somehow.

But perhaps not in the title of the help text, eh?

Reply:
Indeed, it might be misleading to emphasize that the login is restricted, when by default it is not.

Reply:
Very good, I've removed it from the title.